### PR TITLE
Fix java.lang.ClassCastException that occurred in playerSkullsByChargedCreeper rule

### PR DESCRIPTION
**Pull request by keikoutou**
Killing a player or fake player with a /kill command or zombie, etc. when a **playerSkullsByChargedCreeper** rule is true result in a java.lang.ClassCastException.

### CODE REVIEW

File: patches/net/minecraft/entity/player/EntityPlayerMP.java.patch [hunk 1]
```diff
@@ -1,23 +1,47 @@
 --- ../src-base/minecraft/net/minecraft/entity/player/EntityPlayerMP.java
 +++ ../src-work/minecraft/net/minecraft/entity/player/EntityPlayerMP.java
-@@ -1,5 +1,8 @@
+@@ -1,12 +1,26 @@
  package net.minecraft.entity.player;
  
+-import com.google.common.collect.Lists;
+-import com.mojang.authlib.GameProfile;
+-import io.netty.buffer.Unpooled;
```
Comment (Earthcomputer) [inline patches/net/minecraft/entity/player/EntityPlayerMP.java.patch]: Don't rearrange the imports in a patch file

File: patches/net/minecraft/entity/player/EntityPlayerMP.java.patch [hunk 2]
```diff
@@ -129,21 +124,33 @@
...
-+                    ItemStack playerskull = new ItemStack(Items.field_151144_bL, 1, 3);
-+                    playerskull.func_77982_d(JsonToNBT.func_180713_a(String.format("{SkullOwner:\"%s\"}", func_70005_c_().toLowerCase())));
-+                    this.func_70099_a(playerskull, 0.0F);
-+                } catch (NBTException e) {
-+                    e.printStackTrace();
++        if (CarpetSettings.playerSkullsByChargedCreeper)
++        {
++            Entity entity = p_70645_1_.func_76346_g();
++            
++            if (entity != null && entity.getClass().getSimpleName().equals("EntityCreeper"))
```
Comment (Earthcomputer) [inline patches/net/minecraft/entity/player/EntityPlayerMP.java.patch]:
```suggestion
+            if (entity instanceof EntityCreeper)
```

**Review by Earthcomputer — CHANGES_REQUESTED** [review]

**Comment by keikoutou** [comment]
Is this all right?

**Review by Earthcomputer — APPROVED** [review]
Looks good, thanks!

**Pull request merged**